exercises/concept/amusement-park/.docs/instructions.md

```diff
@@ -0,0 +1,140 @@
+# Instructions
+
+It is your job to provide and maintain parts of the IT system of an amusement park. In particular, you are responsible for the program that manages visitors and tickets.
+
+First you will create a new visitor in the system, then you will provide a function for revoking a ticket.
+In task 3 and 4 you will work on the ticket tracking system that helps to prevent forgery.
+Lastly you will help adapting the system to new legal requirements.
+
+## 1. Create a new visitor
+
+When visitors come to the amusement park for the first time, they need to register by entering their name and age in a terminal and agreeing to the terms and conditions.
+Of course they also need to buy a ticket.
+Each ticket has an identifier like `H32AZ123`.
+
+Write a function `createVisitor` that accepts three arguments.
+
+- The name of the visitor.
+- The age of the visitor.
+- The identifier of the ticket that the visitor bought.
+
+The function should return an object that holds this information.
+The keys in the object should be called `name`, `age` and `ticketId`.
+
+```javascript
+createVisitor('Verena Nardi', 45, 'H32AZ123');
+// => { name: 'Verena Nardi', age: 45, ticketId: 'H32AZ123' }
+```
+
+## 2. Revoke the ticket
+
+When people leave the amusement park, their ticket gets revoked. If they come back, they need to buy a new one.
+To save regular visitors some time, they only need to register once and the visitor information will be kept for subsequent visits.
+
+That means when a visitor leaves the park, only their ticket should be invalidated but the rest of the visitor object should stay the same.
+Implement the function `revokeTicket` that accepts a visitor object, sets the ticket identifier to `null` and returns the object afterwards.
+
+```javascript
+const visitor = {
```

Comment:
You use `const` here but `let` in a lot of other places (where `const` could have been used). Maybe consider using `let` here to keep consistency?

Reply:
Or the other way around, make them all `const` unless there is a reassignment (Happens only on 2-3 places).

Unless, students are not that advanced on this stage; not expected to know the difference between `const` and `let`. Then make them all `let`.

Reply:
As far as I am aware I am consistently using `const` everywhere besides the examples for null and undefined in introduction.md/amount.md. `const name;` is not allowed and something like `const name = null` wouldn't make make sense as the text below talks about assigning another value later. That is why I used `let` in these examples.

The student might not fully understand the `let`/`const` difference until he/she gets to the deep dive concept on variables at some point but I think we have to live with that. We just cannot teach everything at once.